### ISSUE TITLE
Flags are wrong

### ISSUE THREAD

**Issue by raphink** [dominikh/go-augeas#6]
```go
	aug, _ := augeas.New("/", "", augeas.None)
	v, _ := aug.Get("/augeas/save")
	log.Println(v)
```

prints `backup`. Using `augeas.Backup` prints `newfile`, etc.

**Comment by raphink** [comment]
As a result, `augeas.None` saves as backup instead of saving to the same file, and using `augeas.SaveNewFile` is very slow because it causes Augeas to typecheck all its lenses.

**Comment by dominikh** [comment]
Closed by 09517007e3a091173c283147529ed1de94b105d1
